content/en/docs/Deployment/affinities-and-tolerations.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Kubernetes Affinities and Tolerations"
+linkTitle: "Affinities and Tolerations"
+weight: 50
+date: 2020-09-04
+description: >
+  Managing which Kubernetes nodes workloads get deployed to
+---
+
+This document describes how the administrator may influence how the various
+workloads that are part of KubeCF will get deployed onto their Kuberntes
+cluster.
+
+## Affinities
+
+Kubernetes will attempt to schedule work based on their [affinities and
+anti-affinities]; by default, KubeCF will request for each instance group to be
+scheduled away from other replicas of the same instance group.  Additionally,
+by default the `router` instance group and the `diego-cell` instance group will
+have an anti-affinity towards each other.
+
+[affinities and anti-affinities]: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+
+The affinities can be overridden on a per-instance-group basis, using helm
+configuration values such as the following:
+
+```yaml
+sizing:
+  uaa:
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+            - key: quarks.cloudfoundry.org/quarks-statefulset-name
+              operator: In
+              values:
+              - uaa
+              - api
+```
+
+This will request `uaa` pods to avoid any other `uaa` or `api` pods; this may be
+helpful if your UAA instances consume resources to the point where they slow
+down API access.  Note that if any affinity or anti-affinity options are given,
+they will override the default anti-affinities; it is recommended that they be
+specified explicitly as well, as given in the example above.
+
+Note that it is also possible to declare `podAffinity`, as the whole affinity
```

Comment:
Maybe mention *both* pod and node affinity? I can't really think of any kubecf use-case where you want to put 2 different instance groups on the same node. However, we do have a use-case for `nodeAffinity`, so that should be mentioned here.

Reply:
Mentioned in 92620359e3ea6703e377cfaa3c6d41e7ab433a46